package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-camera",
   "description": "A Camera component for React Native. Also reads barcodes.",
-  "version": "3.26.0",
+  "version": "3.27.0",
```

Comment:
no need to bump here, the bump is automated

Reply:
Have reverted it back to its original value.